docs/oauth2-examples-auth0.md

```diff
@@ -21,7 +21,8 @@ limitations under the License.
 
 # Use https://auth0.com/ as OAuth 2.0 server
 
-Let's test the following 3 OAuth flows:
+Demonstrate how to authenticate using OAuth 2.0 protocol
+and Auth0 as Authorization Server using the following flows:
 
```

Comment:
Add "the" after "using" and before "OAuth 2.0 protocol"
Add "the" after and "Auth0" and before "as Authorization Server"